src/main/resources/screens/COMP_HIDDEN.json

```diff
@@ -0,0 +1,37 @@
+{
```

Comment:
Screens like these aren't actually required anymore as summary, accordion and extracts all have their own config. Removing this aren't in scope of this change though - as we need to ensure each isn't used _somewhere_...